runtime/service_integ_test.go

```diff
@@ -415,9 +416,12 @@ func TestMultipleVMs_Isolated(t *testing.T) {
 
 						jailer := &runcJailer{
 							ociBundlePath: string(shimDir),
+							vmID:          vmIDStr,
 						}
 						_, err = os.Stat(jailer.RootPath())
 						require.NoError(t, err, "failed to stat root path of jailer")
+						_, err = os.Stat(filepath.Join("/sys/fs/cgroup/cpu", resp.CgroupPath))
```

Comment:
Can we add an assertion that `resp.CgroupPath` is the expected value too? Otherwise I feel like this stat could succeed even if the cgroup path was unexpectedly `""`, `"/"` or something like that

Reply:
Fixed!